src/util/config.ts

```diff
@@ -0,0 +1,8 @@
+import 'dotenv/config';
+
+const config: { HOST: string, PORT: number } = {
+  HOST: process.env.HOST || '127.0.0.1',
+  PORT:  parseInt(`${process.env.PORT}`, 10) || 3001
```

Comment:
You can use the `+` sign to convert directly to number: `+process.env.PORT`.

Reply:
Perfect, thanks, can you explain me why? Is better performance? Or this cause issues minor in the future?

Reply:
Just a comment that you can use that trick to show more skills 😅  and the code will be more readable. In your code, you have to learn how the parseInt function works and what the 10 means.

Reply:
I see! Thanks it's a good trick for have a good code :), thanks!